### ISSUE TITLE
When processing the style attribute，Vue.js miss the -webkit profix

### ISSUE THREAD

**Issue by lcjnil** [vuejs/vue#8612]
### Version
2.5.17

### Reproduction link
[https://jsfiddle.net/k1uptbpw/7/](https://jsfiddle.net/k1uptbpw/7/)

### Steps to reproduce
open the jsfiddle link，and use devtool to inspect the element

### What is expected?
the style on the div element should be "-webkit-background-clip: text;"

### What is actually happening?
the style on the div element is "background-clip: text;"

---
This is a Chrome's bug（my version is 67）, and it will be ok when using "style.cssText" to set the style. And according to Can I Use, when use this property, it must with the prefix "-webkit", so I think there is two way to resolve this:
1.(ugly)change vue's "setProp" method
2.add some case in the official document

**Comment by posva** [comment]
Well, after diving into it, it looks like setting the property using js breaks (using cssText or style.webkitBackgroundClip), it only works setting it with `setAttribute('style')`.
Did you find this reported as a bug in webkit tracker?

**Comment by skyline0705** [comment]
I have found the issue list and there has no issue about this……So I just create  an issue to them: https://bugs.chromium.org/p/chromium/issues/detail?id=872114

**Comment by Justineo** [comment]
@posva I've revisited this issue and it seems that the style settings are actually working though Chrome is providing the wrong value for `style` attribute.

See [this fiddle](https://jsfiddle.net/1zf73b8k/).

When you do

```js
el.style.webkitBackgroundClip = 'text'
// or in Vue.js is actually
el.style.setProperty('-webkit-background-clip', 'text')
```

It actually works!

But after syncing the value back from CSSOM, Chrome got us this:

```js
console.log(el.getAttribute('style')) // 'background-clip: text;'
```

And in the Devtools we also got ~`background-clip: text`~.

While we actually got this:

```js
console.log(el.style.webkitBackgroundClip) // 'text'
```

But if we change the value like this:

```js
el.setAttribute('style', '-webkit-background-clip: text')
```

Everything seems to be working as expected. So it seems to be something wrong within the syncing process between style IDL attribute (`CSSStyleDeclaration`) and content attribtue (`string`).
